softuart.c:41:51: error: initialization makes integer from pointer without a cast [-Werror]

### Expected behavior
dev branch (commit 8c08e76) compiles

### Actual behavior
```text
CC app/modules/softuart.c
softuart.c:41:51: error: initialization makes integer from pointer without a cast [-Werror]
 static task_handle_t uart_recieve_task = NULL;
                                                   ^
cc1: all warnings being treated as errors
make[2]: *** [.output/eagle/debug/obj/softuart.o] Error 1
../../Makefile:422: recipe for target '.output/eagle/debug/obj/softuart.o' failed
make[2]: Leaving directory '/opt/nodemcu-firmware/app/modules'
make[1]: *** [.subdirs] Error 2
../Makefile:380: recipe for target '.subdirs' failed
make[1]: Leaving directory '/opt/nodemcu-firmware/app'
make: *** [.subdirs] Error 2
Makefile:380: recipe for target '.subdirs' failed
make: Leaving directory '/opt/nodemcu-firmware'
```

### Test code
https://github.com/mk-pmb/nodemcu-firmware-daily-vanilla/runs/371915996

### NodeMCU version
dev branch, commit 8c08e76

### Hardware
The Github cloud

--- a/app/modules/softuart.c
+++ b/app/modules/softuart.c
@@ -38,7 +38,7 @@ softuart_t * softuart_gpio_instances[SOFTUART_GPIO_COUNT] = {NULL};
 // Array of callback reference to be able to find which callback is used to which rx pin
 static int softuart_rx_cb_ref[SOFTUART_GPIO_COUNT];
 // Task for receiving data
-static task_handle_t uart_recieve_task = NULL;
+static task_handle_t uart_recieve_task = 0;
 // Receiving buffer for callback usage
 static char softuart_rx_buffer[SOFTUART_MAX_RX_BUFF];
 
@@ -388,8 +388,8 @@ static int softuart_gcdelete(lua_State *L)
 // Port function map
 LROT_BEGIN(softuart_port)
 	LROT_FUNCENTRY( on, softuart_on)
-	LROT_TABENTRY( __index, softuart_port)
 	LROT_FUNCENTRY( write, softuart_write)
+	LROT_TABENTRY( __index, softuart_port)
 	LROT_FUNCENTRY( __gc, softuart_gcdelete)
 LROT_END(ads1115, softuart_port, LROT_MASK_GC_INDEX)
 

